### PR TITLE
Obmin 316 modified form change email

### PR DESCRIPTION
**Pull request by Mish-hash**

### CODE REVIEW

**Review by Viktoria-Serkiz — DISMISSED** [review]

**Review by Voloshkina — APPROVED** [review]

File: src/main/resources/react/src/components/local/en.js [hunk 1]
```diff
@@ -90,9 +90,8 @@ const en = {
     newPassword: 'New password:',
     confirmPassword: 'Confirm password:',
     changeEmail: 'Change EmaiL',
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/local/en.js:92]: Are you sure what this field needs?
you need one fields

File: src/main/resources/react/src/components/local/ua.js [hunk 2]
```diff
@@ -90,8 +90,8 @@ const ua = {
     newPassword: 'Новий пароль:',
     confirmPassword: 'Повторіть пароль:',
     changeEmail: 'Змінити EmaiL',
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/local/ua.js:92]: the same

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_settings/config.js [hunk 3]
```diff
@@ -43,24 +43,16 @@ export const validationPasswordSchema = yup.object().shape({
...
-export const validationEmailSchema = (currentEmail) =>
+export const validationEmailSchema = (email) =>
   yup.object().shape({
-    newEmail: yup
+    email: yup
       .string()
       .required(getTranslatedText('errors.requireField'))
       .matches(NO_SPACE, getTranslatedText('errors.noSpace'))
-      .notOneOf([currentEmail], getTranslatedText('errors.emailIdentical'))
+      .notOneOf([email], getTranslatedText('errors.emailIdentical'))
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_settings/config.js:52]: I think this field don't need

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_settings/index.jsx [hunk 4]
```diff
@@ -233,41 +227,16 @@ const MySettings = () => {
...
-                inputJustifyContent="space-between"
-                label={getTranslatedText('settings.oldEmail')}
-              />
-
               <InputField
                 type="text"
-                name="newEmail"
+                name="email"
                 labelColor="black"
+                value={values.email}
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_settings/index.jsx:234]: check pls
this field don't need

**Review by Wolshebnik — COMMENTED** [review]
We should to meet

**Review by Kyslytsia — APPROVED** [review]

**Review by ASGladchenko — DISMISSED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Review by Viktoria-Serkiz — APPROVED** [review]

**Review by cloudmenson — APPROVED** [review]

**Review by Kyslytsia — APPROVED** [review]

**Review by Voloshkina — APPROVED** [review]

**Pull request merged**